pkg/validation/validate_cluster.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/tools/pager"
+	kopsapi "k8s.io/kops/pkg/apis/kops"
```

Comment:
This should already be imported as kops below, I believe

Reply:
yes @justinsb will be doing this.